unlock-app/src/components/interface/keychain/CancelAndRefundModal.tsx

```diff
@@ -73,6 +73,7 @@ export const CancelAndRefundModal = ({
       lockAddress,
       tokenId,
     }
+    const walletService = await changeNetwork(network)
```

Comment:
What do you think of actually naming this `getWalletServiceFor(network)`  and just using it everywhere in the app when we need a walletService?